### PR TITLE
[#GEA] Implementing grid based map

### PR DESCRIPTION
**Pull request by lemara98**
I have changed the entire map template to allow the changes on the map to only specific places on the map.

### CODE REVIEW

File: GraphicEditor/MainWindow.xaml.cs [hunk 1]
```diff
@@ -16,10 +16,16 @@ public partial class MainWindow : Window
...
         public static double MainScreenHeight = SystemParameters.PrimaryScreenHeight * SCREEN_FACTOR;
 
+        
+
         public MainWindow()
         {
             Application.Current.MainWindow.Width = MainScreenWidth;
             Application.Current.MainWindow.Height = MainScreenHeight;
+
+            //Application.Current.MainWindow.MinWidth = 800;
```
Comment (lukaDoric) [inline GraphicEditor/MainWindow.xaml.cs]: Delete unnecessary code 🗡️

**Review by lukaDoric — CHANGES_REQUESTED** [review]

**Review by Ana00000 — APPROVED** [review]

File: GraphicEditor/MainWindow.xaml.cs [hunk 2]
```diff
@@ -16,10 +16,13 @@ public partial class MainWindow : Window
         public static double MainScreenWidth = SystemParameters.PrimaryScreenWidth * SCREEN_FACTOR;
         public static double MainScreenHeight = SystemParameters.PrimaryScreenHeight * SCREEN_FACTOR;
 
+        
+
```
Comment (milicasiriski) [inline GraphicEditor/MainWindow.xaml.cs:20]: extra empty lines

**Review by milicasiriski — COMMENTED** [review]
There's not much code, mostly xaml 😄 so let's just wait for your teammates to check if everything is according to guidelines

**Review by lukaDoric — APPROVED** [review]

**Review by VladimirBudjen — APPROVED** [review]

**Comment by Ana00000** [comment]
[(1) Implementing grid based map [1]](https://trello.com/c/KhQxETHK/42-1-implementing-grid-based-map-1)

**Pull request merged**